holmes/plugins/toolsets/grafana/toolset_grafana_tempo.py

```
@@ -291,14 +321,245 @@
+                    )
+                    response.raise_for_status()
+                    return {
+                        "traceID": trace_id,
+                        "durationMs": trace_summary.get("durationMs", 0),
+                        "rootServiceName": trace_summary.get(
+                            "rootServiceName", "unknown"
+                        ),
+                        "traceData": response.json(),  # Raw trace data
+                    }
+                except requests.exceptions.RequestException as e:
+                    error_msg = f"Failed to fetch full trace: {str(e)}"
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)